.ci/jobs/package-registry.yml

```diff
@@ -0,0 +1,23 @@
+---
+---
```

Comment:
@kuisathaverat will the double separator fail the YAML parser?

Reply:
yep https://github.com/elastic/beats/pull/18881

Reply:
Oh I see... I approved the PR and then found this... which was the same 🤦